containers/sidebar/StorageSpaceStatus.js

```diff
@@ -12,30 +12,33 @@ const StorageSpaceStatus = ({ children }) => {
     const usedPercent = Math.round(UsedSpace / MaxSpace);
     const maxSpaceFormatted = humanSize(MaxSpace);
     const usedSpaceFormatted = humanSize(UsedSpace);
-    const color = usedPercent < 60 ? 'global-success' : usedPercent < 80 ? 'global-attention' : 'global-warning';
+    const color =
+        usedPercent < 60
+            ? 'circle-bar--global-success'
+            : usedPercent < 80
+            ? 'circle-bar--global-attention'
+            : 'circle-bar--global-warning';
```

Comment:
if we're using cicle-bar prefix, maybe it would be better to name status to low/medium/full instead of color names? Feels a bit redundant otherwise.

Reply:
Yes, good idea. :)

As I'm waiting some design elements (icon for the "i"), I'll rename them at the same time when I will update it :)

I've created an issue to avoid forgetting it https://github.com/ProtonMail/design-system/issues/228